examples/values-sigstore-openshift.yaml

```diff
@@ -58,7 +58,8 @@ configs:
   cosign:
```

Comment:
this file will be much smaller - as many of these are now set in the default values file with this PR

Reply:
updated